homeassistant/components/shelly/binary_sensor.py

```diff
@@ -48,13 +53,31 @@
     ),
 }
 
+REST_SENSORS = {
+    ("cloud"): RestAttributeDescription(
```

Comment:
You can omit the parentheses.

```suggestion
    "cloud": RestAttributeDescription(
```